nixos/modules/system/activation/bootspec.nix

```diff
@@ -19,13 +19,15 @@ let
           (builtins.toJSON
           {
             v1 = {
+              system = config.boot.kernelPackages.stdenv.hostPlatform.system;
```

Comment:
I don't know of the proper way, but I really don't like this way of getting the system :'(

(Unless this is how it's done elsewhere in Nixpkgs...?)

Reply:
nvm, this is fine; that's how `$toplevel/system` gets generated :(

Reply:
Correct :D.

Maybe, @roberth or @Ericson2314 may have better ideas about this.

Reply:
Anything that isn't `config.nixpkgs.*system` is good, because those were trash.
I would've been happy with `pkgs.stdenv.hostPlatform.system`, but taking it from the kernel is perhaps more accurate? Anyway, lgtm.